### PR TITLE
Prettify

### PR DESCRIPTION
**Pull request by ThomasITU**
Minor refactor, with very pretty print 🎉 
image

### CODE REVIEW

**Comment by simonskodt** [comment]
Looks, nice, have you tried running our solution against all the `.in` and `.out` files?

File: gorilla.py [hunk 1]
```diff
@@ -17,23 +15,86 @@ def main():
...
+            drop_j = HYPHEN_COST + alignment[row][col-1]
+            alignment[row][col] = max(take, drop_i, drop_j)
+
+    return alignment[len(dna_sequence_y)][len(dna_sequence_x)], backtrack(alignment, dna_sequence_x, dna_sequence_y)
+
+def backtrack(alignment, dna_sequence_x, dna_sequence_y):
+    global HYPHEN_COST
+
+    backtrack_dna_sequence_x = ""
+    backtrack_dna_sequece_y = ""
```
Comment (Faurby) [inline gorilla.py]: sequence instead of sequece

**Review by Faurby — COMMENTED** [review]
Looks very good! Just a minor comment on a spelling error

**Review by PhilipFlyvholm — APPROVED** [review]

**Review by simonskodt — APPROVED** [review]

File: gorilla.py (hunk 1 above)
Comment (ThomasITU) [reply gorilla.py]: Good catch!

**Pull request merged**